apps/docs/content/components/application/HopperProvider.mdx

```diff
@@ -16,21 +16,21 @@ links:
 ## Application provider
 
 A HopperProvider must be the root component of your application. All other Hopper components rely on the Provider to define the color scheme, locale, and other settings that they need in order to render.
-You can also nest multiple HopperProviders to create different themes or locales for different parts of your application.
+You can nest multiple HopperProviders to create different themes or locales for different parts of your application if needed.
 
 ### Color scheme
 
-We recommend supporting both light and dark color schemes in your app, however, if you need to override this with an application specific setting, you can use the colorScheme prop.
+We recommend supporting both light and dark color schemes in your application, however, if you need to override this with an application specific setting, you can use the colorScheme prop.
 
 <CodeOnlyExample src="HopperProvider/docs/hopper-provider/color-scheme" />
 
-See the [styling documentation](/components/concepts/styling) for more information about using Hopper color variables in your  app to ensure it adapts to light and dark mode properly.
+See the [styling documentation](/components/concepts/styling) for more information about using Hopper color variables in your application to ensure it adapts to light and dark mode properly.
 
 See the [color schemes documentation](/components/concepts/color-schemes) for more information on how to apply a color scheme to your application.
 
 ### Locales
 
-Another important setting for your application is the locale. By default, Hopper chooses the locale matching the user's browser/operating system language, but this can be overridden with the locale prop if you have an application specific setting. This prop accepts a [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt) language code.
+Another important setting for your application is the locale. By default, Hopper chooses the locale matching the user's browser/operating system language, this can be overridden with the locale prop if you have an application specific setting. This prop accepts a [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt) language code.
```

Comment:
```suggestion
Another important setting for your application is the locale. By default, Hopper chooses the locale matching the user's browser/operating system language. This can be overridden with the locale prop if you have an application specific setting. This prop accepts a [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt) language code.
```
Usually if you have a comma, you would use which and not this. If you have this, then it should most likely be a new sentence. yes, I know it was already there :P